lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -932,10 +792,23 @@ void LCIStrategicPlugin::planWhenAPPROACHING(const cav_srvs::PlanManeuversReques
 
   ROS_DEBUG_STREAM("SPEED PROFILE CASE:" << ts_params.case_num);
 
-  // CASE SELECTION END
+  /////////////  4. Safety Check against traffic signals and Final Maneuver Generation //////////////
+
+   double safe_distance_to_stop = pow(current_state.speed, 2)/(2 * max_comfort_decel_norm_);
+  ROS_DEBUG_STREAM("safe_distance_to_stop at max_comfort_decel:  " << safe_distance_to_stop << ", max_comfort_decel_norm_: " << max_comfort_decel_norm_);
+
+  double desired_distance_to_stop = pow(current_state.speed, 2)/(2 * max_comfort_decel_norm_ * config_.deceleration_fraction) + config_.desired_distance_to_stop_buffer;
```

Comment:
Should this be absolute value'd, just in case or is max_comfort_decel_norm_ guaranteed to be always >0?

Reply:
Yeah it is guaranteed to be always >0 because the name also suggests it is the "norm"